src/App.js

```diff
@@ -17,12 +17,22 @@ function App() {
 
   // dropdownArea: arrow function that returns the apporpriate button (and dropdown menu if applicable) depending on whether dropdownOpen is true/false
   const dropdownArea = () => {
-    return (
-      <button className='button' onClick={dropdownHandler}>
-        <div className='triangle' {...dropdownOpen ? "up" : ""}></div>
```

Comment:
Oops, I should've caught this on the second CR. So here, we want to use string interpretation:

```
<div className=`triangle ${dropdownOpen ? "up" : ""}`></div>
```

We're using backticks (the `) so we can use the in-string `${}` thing (I'm not exactly sure what this is called, haha). Then, we have a ternary operator that returns "up" if the dropdown is open, and nothing if it's closed.

Reply:
<img width="883" alt="Screen Shot 2020-09-18 at 1 55 05 PM" src="https://user-images.githubusercontent.com/50722281/93644353-abce2780-f9b6-11ea-9c13-0224fad1e3d7.png">

This is how the code looks in the editor; there's a bunch of errors that come up

Reply:
Oops sorry, it should be 

```jsx
<div className={` ... `}></div>
```

My bad!